include/benchmark/benchmark.h

```diff
@@ -430,10 +430,50 @@ struct Statistics {
       : name_(name), compute_(compute) {}
 };
 
+// Converts whatever the stored manual time is to
+// seconds, which is used to estimate iterations
+// and similar
+typedef double(BenchmarkTimeCostFunc)(double);
+// default assumes measurements are done in seconds
+// (as they are currently done in the benchmark runner)
+inline double DefaultBenchmarkTimeCostFunc(double measurement) {
+  return measurement;
+}
+
 struct BenchmarkInstance;
 class ThreadTimer;
 class ThreadManager;
 
+class BenchmarkTime {
```

Comment:
@dominichamon isn't `BenchmarkTime` a little bit too vague?
How do i know if it is not the actual time, but actually a `BenchmarkTimerDsc`?
How about using the latter name?

Reply:
(i'll be ok with either `BenchmarkTimerDsc` or `BenchmarkTimeDsc`, just not with `BenchmarkTime`.)

Reply:
I went with `Info` instead of `Dsc`, because it's a bit easier to read.